model-test/src/main/java/org/jboss/as/model/test/ModelTestModelDescriptionValidator.java

```diff
@@ -113,6 +115,8 @@
         validChildTypeKeys.put(MIN_OCCURS, SimpleIntDescriptorValidator.INSTANCE);
         validChildTypeKeys.put(MAX_OCCURS, SimpleIntDescriptorValidator.INSTANCE);
         validChildTypeKeys.put(ALLOWED, StringListValidator.INSTANCE);
+        validChildTypeKeys.put(FILESYSTEM_PATH, BooleanDescriptorValidator.INSTANCE);
```

Comment:
nothing, i wrongly read it above and tough it was a string and not a boolean. 
sorry for confusion